src/test/java/javaslang/collection/AbstractTraversableOnceTest.java

```diff
@@ -1167,6 +1165,40 @@ public void shouldHandleStdoutIOException() {
         }
     }
 
+    // -- PrintStream
+
+    @Test
+    public void shouldWriteToPrintStream() {
+        final ByteArrayOutputStream baos = new ByteArrayOutputStream(1024);
+        final PrintStream out = new PrintStream(baos);
+        of(1, 2, 3).out(out);
+        assertThat(baos.toString()).isEqualTo(of(1, 2, 3).mkString("\n", "", "\n"));
```

Comment:
we should instantiate <code>of(1, 2, 3)</code> twice because of <code>Iterator</code> behaviour